src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -943,6 +943,11 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
 
                     body = BindMethodBody(methodSymbol, compilationState, diagsForCurrentMethod, out importChain, out originalBodyNested);
 
+                    if (body != null && methodSymbol.MethodKind == MethodKind.Constructor)
```

Comment:
Should we do the same for static constructors?

Reply:
Yes, we should handle static constructors and fields, but in a later PR. There are `// PROTOTYPE` comments in the unit tests.